app/components/code-sample/template.hbs

```diff
@@ -0,0 +1,8 @@
+<div class="code-sample" data-uri="{{ default _ref self }}" >
+    <div data-editable="code">
+        {{#if html}}
+           {{!-- code tag should be in the same line as pre tab, i will break the style if not --}}
```

Comment:
I think there's a typo in here
```suggestion
           {{!-- code tag should be in the same line as pre tab, it will break the style if not --}}
```

Reply:
done